app/views/admin/conference/edit.html.haml

```diff
@@ -23,4 +23,13 @@
         = f.input :end_date, :as => :string, :input_html => { :id => "conference-end-datepicker", :readonly => "readonly"  }
       = f.inputs name: "Registrations" do
         = f.input :registration_limit, as: :number, in: 0..9999, hint: "Limit the number of registrations to the conference (0 no limit)"
+      = f.inputs name: "Tickets" do
```

Comment:
We use [Bootstrap-Switch-Checkbox](https://github.com/openSUSE/osem/wiki/Bootstrap-Switch-Checkbox) for this kind of input